site/content/en/guides/consumer/display/_index.md

```diff
@@ -163,7 +163,6 @@ will print arbitrary fields by providing the `--field` flag.
 
 ### Filter Command
 
-<!-- @filterExamples @verifyGuides-->
```

Comment:
why is this going away?

Reply:
That block returns an error code (in the verifyGuides workflow) with the version of `kpt` built from this branch but not with 0.39.2 locally. There may have been some breakage in the behavior since that release.